content/posts/2021/01/2021-06-14-outreachy-application-process/index.md

```diff
@@ -0,0 +1,28 @@
+hero: ./images/outreachy.png
+excerpt: This blog outlines my Outreachy application process and some personal advise about the same.
+---
+
+# My Outreachy Application Process
+
+Outreachy provides a 3 month paid internship to work in Free and Open Source Software. It happens twice each year(in Summer and Winter) and targets people underrepresented in tech.I participated in the Outreachy Internship program during the 2018 Winter round.
+
+I first heard of Outreachy from a friend who had previously participated in the program. The reason why I decided to apply to  the program is because  I love open source and would like to work in open source in future, so Outreachy was a great place to get started and the experience would be good for my career generally. I signed up and went through the information on [the website](https://www.outreachy.org/) about the internship months before the applications were opened. I also signed up for a newsletter which informs you when the applications open. I went through past projects and organizations which gave me an idea of some the organizations which I would like to apply to. There are blogs from past interns talking about their application processes and they also offer tips on what worked based on their experiences. I read some of these which gave me an idea on what to expect. When the internship applications opened, I got an email telling me so :mailbox:
+
+The first step of the application process is the initial/eligibility application. This consists of a number of questions which help Outreachy organizers determine if you are eligible for Outreachy mainly based on on your availability during the internship period and if you are underreprented in tech spaces in your area.  I submitted my initial application had to wait a few days as my proposal was reviewed and decided on if I was eligible for the program. After I got the go ahead, I looked through the projects  to find the ones I could apply to. I selected three projects that I had interest on. I started looking in the code base and entered their communication channels to find more information about the projects. Other projects were still being added to the website and about two weeks in, I saw that Public Lab plots2 project was added in the list of projects.
+
+I had an experience with Public Lab community before, I researched further and totally resonated with what they do. Public Lab is a community and non-profit democratizing science to address environmental issues that affect people. It is a platform where you can post your research, open source it for others to use and also find information you need to solve your environmental and social injustices. There is so much more on [Public lab](http://publiclab.org/). Plus the code community is welcoming and ready to help. I love what Public Lab does and after I saw it listed I knew that is exactly what I would like to work on so I focused on it. You are required to pick 1-2 projects and apply to.
+
```

Comment:
@cesswairimu Would be nice if you add a sub-heading here eg...
Eligibility  
To get the reader's attention as well

Reply:
mmh not sure...may be misguiding...I am mentioning one eligibility criteria here(there are others) and then continuing with my process...but if you think it makes senses to add that am glad to it. thanks

Reply:
Cool got it, In that case then you can add also slightly mention that in this blog you will focus on one eligibility criteria that worked well for you and that there are other more criteria that they could check as well/qualify for.